sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClientBuilder.java

```diff
@@ -124,7 +124,7 @@ public ServiceBusAdministrationAsyncClient buildAsyncClient() {
             .apiVersion(apiVersion.getVersion())
             .buildClient();
 
-        return new ServiceBusAdministrationAsyncClient(client, serializer);
+        return new ServiceBusAdministrationAsyncClient(client, serializer, tokenCredential);
```

Comment:
Please ignore this file - got included due to bad git branch structuring - will be fixed as soon as the 25146 PR is merged, this will become clean

Reply:
should we hold off on this PR until the other one is merged and this is rebased?

Reply:
adding `Do Not Merge` label to denote that there's some git cleanup necessary.  Please remove the label when the PR is ready :)